style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!===` to `==` and `!=`.
+* Avoid using arrow functions when not necessary.
```

Comment:
> Not a technical problem, but I like being able to visually distinguish the two.

This sounds a lot like the discussion around single vs double quotes. If we can default to always using the fat arrow and never have to come back and change it, why not? I don't think the readability actually changes that much.

Are there any technical downsides to using the fat arrow everywhere?